docs/OCP/StartingANewProject.md

```diff
@@ -4,52 +4,28 @@ description: The process for getting a new project set up on the BC Gov OpenShif
 ---
 ## How to Request a New OpenShift Project
 
-The work of different teams within the OpenShift platform is organized into isolated "projects".  Prior to a team beginning work on the platform, they must have a set of projects provisioned for them.  Project provisioning for the OpenShift platform is managed by the OCIO Enterprise DevOps branch. This document describes the pre-requisites as well as the mechanics of the provisioning process. 
+The work of different teams within the OpenShift platform is organized into isolated "projects".  Prior to a team beginning work on the platform, they must have a set of projects provisioned for them by the OCIO Enterprise DevOps branch. This document describes the pre-requisites and mechanics of the provisioning process. 
+
+***Each project must be reviewed and accepted, including additional projects for a team that already has project(s) on the platform.*** 
 
 ## Pre-requisites
 
-Prior to project provisioning, a manual/human step is required that involves a project/product team lead / sponsor communicating with the Enterprise DevOps branch leadership and having their project/app/product reviewed and accepted for deployment on the DevOps platform.
+Prior to project provisioning, your team lead/sponsor will need to meet with the Enterprise DevOps branch leadership to have your project/app/product reviewed and accepted for deployment on the DevOps platform.
 
-The projects currently deployed on the platform are largely "Continuous Service Improvement Lab"-affiliated, but there is a set of other well-aligned projects on the platform as well.
```

Comment:
Suggest deleting this as it may only cause the reader to question whether their project is suitable or not. By removing it, I think it will help them just focus on the **Criteria for acceptance** below.